.nvmrc

```diff
@@ -0,0 +1 @@
+lts/hydrogen
```

Comment:
Could you exclude this file from your PR (or/and add it to `.gitignore`)?

That being said, in a separate PR, it might be a good idea to reintroduce it, while modifying the Github Actions files of this project to also use this same configuration file.

Reply:
I'll happily do so. I think it's important to have since it provides (often near-automatic) setup information before trying to install.

Reply:
You're right, it's a free improvement. Let's keep it here.

In the future, using this config file into the Github Actions files would be a valuable addition (contributions are welcome for an upcoming PR).

Reply:
for sure! I'll do so posthaste

Reply:
@emagnier would the preference be to use the version that volta uses as well? it looks like volta pins to 20.8.1 right now. should it be pinning to an LTS?

Reply:
Good point. An LTS version is probably preferable.

Reply:
cool. i'll make sure i can pin them both. would it be problematic to _just_ use the nvmrc and not the volta config in the package.json?

i think volta supports .nvmrc (i'll submit a PR and i guess we can see?) or i can remove the .nvmrc and put a readme entry about using volta, then i can switch the CI build to use volta, if that's the desired source of truth.

Reply:
actually, I would lean towards droping Volta and relying solely on `.nvmrc` as the source of truth. I haven't been using Volta for some time now (it doesn't support PNPM, which I've been using more and more across various projects), and I believe NVM is much more mature and widely used than Volta.